index.js

```diff
@@ -2382,28 +2382,24 @@
     impl: fromRight
   };
 
-  //# fromEither :: b -> Either a b -> b
+  //# fromEither :: Either a a -> a
   //.
-  //. Takes a default value and an Either, and returns the Right value
-  //. if the Either is a Right; the default value otherwise.
+  //. Takes an Either with the same type on the Left and on the Right
+  //. and return whichever value exists.
```

Comment:
I just spotted a typo:

```suggestion
  //. and returns whichever value exists.
```